srv/salt/_modules/fs.py

```diff
@@ -109,7 +109,7 @@ def _ceph_is_down():
     retries = 6
     delay = 2
     down = False
-    # Processes that don't impede migration (httpd-prefork == openattic, rest are ganesha related).
+    # Processes that don't impede migration (ganesha related).
     omit_list = ['httpd-prefork', 'ganesha.nfsd', 'rpcbind', 'rpc.statd']
```

Comment:
Can we remove httpd-prefork from this list?

Reply:
right, httpd-prefork should be excluded now.. Let me check that again though